### DISCUSSION TITLE
Running & Adding codeql-analysis for all the branches of a repository

### DISCUSSION THREAD

**Discussion by adithyaakrishna** [community/community#26569] [Actions]
<p>Hey GitHub Community,<br>
I wanted to know if running codeql-analysis on all the branches of a repo possible or not.<br>
The action should basically run on all the branches a PR is targeted to from the Main repository. I tried searching across github but couldn’t find anything.<br>
Any help would be very much appreciated<br>
<img src="https://user-images.githubusercontent.com/23498248/181098514-f3640bf8-ba2a-41b8-b485-7df65925ff8a.png" alt="image" data-base62-sha1="zE2A6C2gWhwXihwT2KF4R9Mo4ga" width="645" height="362"></p>

**Comment by kingthorin** [comment]
<p>Your asterisks need to be quoted <code>’**’</code></p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet" target="_blank" rel="noopener nofollow ugc">Workflow syntax for GitHub Actions - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/actions/reference/workflow-syntax-for-github-actions</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by adithyaakrishna** [comment]
<p>Thanks a lot <a class="mention" href="/kingthorin">@kingthorin</a><br>
Also should the square brackets be there or not along with the quotes and **</p>

**Comment by kingthorin** [comment, marked as answer]
<p>Since its a single entry I don’t think they’re necessary. I believe you want something like:</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - '**'
</code></pre>

**Comment by adithyaakrishna** [comment]
<p>Okay, got it. What if I just give it like the below image?</p>
<p><img src="https://user-images.githubusercontent.com/23498248/181098515-98fd136b-2a2f-409d-9b1b-bdb72b4f3fed.png" alt="image" data-base62-sha1="sjfL9ogYfONyXRWvQfXTeqBtXPc" width="370" height="185"></p>
<p>Would that work for my case too?</p>

**Comment by kingthorin** [comment]
<p>I’d have to double check the docs, but I believe so.</p>
<p>Silly me hadn’t even considered not specifying it as I was trying to stick close to your original :man_shrugging:</p>

**Comment by adithyaakrishna** [comment]
<p>Haha, I get that. Thanks a lot <a class="mention" href="/kingthorin">@kingthorin</a> :tada:</p>
